### PR TITLE
New xpdstack and add tomo to xpd

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@mrakitin ready for review!

File: recipes-tag/28-id-2-xpd-collection/meta.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% set year = "2018" %}
 {% set cycle = "2" %}
-{% set version ="0" %}
+{% set version ="4" %}
```
Comment (licode) [inline recipes-tag/28-id-2-xpd-collection/meta.yaml]: 28-id-2 is not on cycle 3, same as 28-id-1?

Comment (CJ-Wright) [reply recipes-tag/28-id-2-xpd-collection/meta.yaml]: Hmm that must not have been updated?

**Comment by licode** [comment]
LGTM.

**Comment by CJ-Wright** [comment]
Please merge when ready

**Pull request merged**